docs/test/anchors.test.js

```diff
@@ -0,0 +1,23 @@
+var fs = require('fs');
+const cheerio = require('cheerio');
+
+const $ = cheerio.load(fs.readFileSync('_site/index.html'));
+
+function escapeRegExp(string) {
+  return string.replace(/[.*+?^$%&{}()|[\]\\]/g, '\\$&'); // $& means the whole matched string
+}
+
+describe('each anchor link should have a corresponding anchor', () => {
+  $('a').each(function (i, link) {
+    const href = $(link).attr('href');
+    // link must include # and must not be an external link
+    if (href.includes('#') && !href.includes('https://') && !href.includes('http://')) {
```

Comment:
If we’re only testing anchors on the same page, maybe filter out by `if (href.startsWith('#')` only?

Reply:
Some links uses `/#` for some reason